src/net/sourceforge/kolmafia/utilities/ChoiceUtilities.java

```diff
@@ -134,43 +149,14 @@ public static Map<Integer, String> parseChoices(final String responseText) {
       Matcher textMatcher = TEXT_PATTERN1.matcher(form);
       String text =
           !textMatcher.find()
-              ? "(secret choice)"
-              : textMatcher.group(1) != null
-                  ? textMatcher.group(1)
-                  : textMatcher.group(2) != null
-                      ? textMatcher.group(2)
-                      : textMatcher.group(3) != null ? textMatcher.group(3) : "(secret choice)";
-      rv.put(key, text);
-    }
-
-    m = LINK_PATTERN.matcher(responseText);
-    while (m.find()) {
-      String form = m.group();
-      if (!form.contains("choice.php")) {
-        continue;
-      }
-      Matcher optMatcher = OPTION_PATTERN2.matcher(form);
```

Comment:
`LINK_PATTERN` uses `OPTION_PATTERN2`, not `OPTION_PATTERN1`

Reply:
ohh... i miss it thank you! i will change it ASAP